airflow/www/views.py

```diff
@@ -336,6 +337,7 @@ def get_int_arg(value, default=0):
 
         state_color_mapping = State.state_color.copy()
         state_color_mapping["null"] = state_color_mapping.pop(None)
+        state_color_mapping.update(STATE_COLORS)
```

Comment:
Isn't this already done from https://github.com/apache/airflow/pull/9520/files#diff-e198cdb4d2a7b4801616a067e6cbd82eR80

Reply:
Updated

Reply:
in https://github.com/apache/airflow/pull/9520/commits/f9601c21dade06bd737af95ed09c5bd946c3b722